sagemaker-clarify/online_explainability/tabular/tabular_online_explainability_with_sagemaker_clarify.ipynb

```diff
@@ -0,0 +1,1051 @@
+{
```

Comment:
<p>Already uncommented... so just rephrase that this will delete the resources....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
[aaronmarkham](https://github.com/aaronmarkham) I did not understand this comment, could you please elaborate here?

Reply:
"## Cleanup\n",
    "\n",
    "Once done with the experiments, **uncomment** and run the below cells to delete the resources."

Reply:
Got it, thanks. I have updated the text in both the notebooks.